src/Components/OwnerInformation.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import '../Styles/OwnerInformation.scss';
+
+const OwnerInformation = ({
+  ownerName,
+  ownerStory, 
+  setOwnerInfo,
+  petName
+}) => {
+
+  return (
```

Comment:
I think it's very clean how we made the owner story it's own component, so that it could be a lot more modular.